payments_config/products.py

```diff
@@ -8,11 +8,14 @@
         'name': _('Mozilla Concrete'),
         'url': 'http://pay.dev.mozaws.net/',
         'terms': 'http://pay.dev.mozaws.net/terms/',
+        'kind': 'products',
```

Comment:
we don't yet have a client who mixes donations and products, so sticking it on the client to avoid duplication works for me